master/buildbot/steps/trigger.py

```diff
@@ -129,7 +141,17 @@ def getSchedulerByName(self, name):
     # This customization enpoint allows users to dynamically select which
     # scheduler and properties to trigger
     def getSchedulersAndProperties(self):
-        return [(sched, self.set_properties) for sched in self.schedulerNames]
+
+            schedulers_and_props = []
+            for sched in self.schedulerNames:
+                d = {
```

Comment:
is there a reason to introduce this temporary variable?  its introduction does not seem to help anything.